sippy-ng/src/component_readiness/CompReadyUtils.js

```diff
@@ -291,7 +295,7 @@ export function formatLongDate(aLongDate, aDateFormat) {
     console.log('Error: unknown date format: ', typeof aLongDate)
     dateObj = new Date(aLongDate)
   }
-  return format(dateObj, aDateFormat)
+  return formatInTimeZone(dateObj, 'UTC', aDateFormat)
```

Comment:
This is a little dicey, but essentially we'd give the function a string like 2024-08-23T00:00:00Z

new Date() would parse that and if you printed it you'd get a date in your local time, i.e. on the 22nd but -3 for my timezone. 

then format would trim that to 2024-08-22T00:00:00 resulting in a control using a day before we told it to.

This should fix it and I *think* the UI is now behaving as you'd expect?

Reply:
Naturally caused problems. Seems the default date when you load in initially and get the default view is (a) not coming from the view, and (b) now today + 1 day somehow. Possible no controls are populated from default view on initial load.